### PR TITLE
Update dark-sites.config

### PR DESCRIPTION
**Pull request by 0PandaDEV**
please add these websites they support dark mode and with dark reader its messed up

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -942,3 +942,5 @@ zoomquilt.org
 zoomquilt2.com
 ztdp.ca
 zunivers.zerator.com
+essentialsp.tk
+pandadev.tk
```
Comment (Gusted) [inline src/config/dark-sites.config:946]: Please order the sites in the list.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dark-sites.config (hunk 1 above)
Comment (0PandaDEV) [reply src/config/dark-sites.config:946]: What do you mean?

Comment (0PandaDEV) [reply src/config/dark-sites.config]: i updated it

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -870,6 +872,7 @@ wago.io
 wallhaven.cc
 wallpaperplay.com
 wannabuy.biz
+waradu.tk
```
[the remark this answers is not in the document]
Comment (0PandaDEV) [reply src/config/dark-sites.config]: actually what means WIP?

**Review by Gusted — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**